public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -182,26 +146,25 @@ const getWelcomeHeaderStyles = (theme: GrafanaTheme2) => ({
     color: theme.colors.text.secondary,
     paddingBottom: theme.spacing(2),
   }),
-  ctaContainer: css`
-    padding: ${theme.spacing(4, 2)};
-    display: flex;
-    gap: ${theme.spacing(4)};
-    justify-content: space-between;
-    flex-wrap: wrap;
-
-    ${theme.breakpoints.down('lg')} {
-      flex-direction: column;
-    }
-  `,
-
-  separator: css`
-    width: 1px;
-    background-color: ${theme.colors.border.medium};
+  ctaContainer: css({
+    padding: theme.spacing(2),
+    display: 'flex',
+    gap: theme.spacing(4),
+    justifyContent: 'space-between',
+    flexWrap: 'wrap',
+
+    [theme.breakpoints.down('lg')]: {
+      flexDirection: 'column',
+    },
+  }),
+  separator: css({
```

Comment:
There is a `Divider` component in `@grafana/ui` but I tried using that instead and it needed other changes to make it behave in flex, and the spacing was off. Again, I didn't want to overdo it in this PR